.travis.yml

```diff
@@ -1,21 +1,42 @@
+# Copyright (c) 2017-2019 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
 language: python
 python:
         - 2.7
         - 3.6
-dist: trusty
+dist: xenial
+addons:
+  apt:
+    packages:
+      - openjdk-8-jre-headless
+      - ant
 matrix:
   include:
     - python: 3.7
-      dist: xenial
       sudo: true
 cache: pip
 
+before_install:
+  # Note: Change next line once support scripts are fixed
```

Comment:
Specifically, this depends on whether SpiNNakerManchester/SupportScripts#17 has been accepted

Reply:
I'll fix once the PR it depends on is sorted. ***Do not accept this PR before then!***